app/change_sets/hyrax/batch_upload_change_set.rb

```diff
@@ -0,0 +1,63 @@
+module Hyrax
+  class BatchUploadChangeSet < WorkChangeSet
+    # include HydraEditor::Form::Permissions
+
+    self.exclude_fields = [:title, :resource_type]
+
+    attr_accessor :payload_concern # a Class name: what is form creating a batch of?
+
+    def self.work_klass
+      ::BatchUploadItem
+    end
+
+    autocreate_fields!
+
+    # On the batch upload, title is set per-file.
+    def primary_terms
+      super - [:title]
+    end
+
+    # # On the batch upload, title is set per-file.
+    # def secondary_terms
+    #   super - [:title]
+    # end
+
+    # Override of ActiveModel::Model name that allows us to use our custom name class
+    # def self.model_name
```

Comment:
I think we want to retain this for routing purposes, but it's unclear if it will be needed or not.